### ISSUE TITLE
Lab 7B no longer works

### ISSUE THREAD

**Issue by rlopezgarcia** [MicrosoftLearning/DP100#40]
This Lab a couple od weeks ago worked perfectly.

Now, in the block of code:
from azureml.pipeline.core import Pipeline

pipeline = Pipeline(workspace=ws, steps=[parallelrun_step])
pipeline_run = Experiment(ws, 'batch_prediction_pipeline').submit(pipeline)
pipeline_run.wait_for_completion(show_output=True)

I left it running for almost 4 hours and does not finish.

The log does not advance

**Comment by nish112** [comment]
I was facing a lot of issues in lab 7b. I am attaching a file, it did the trick for me. You can upload the file in your jupyter environment and it will work like a charm. Currently, there are some bugs in DP100 lab, MS should pay attention to it
[Solutionto7B.zip](https://github.com/MicrosoftLearning/DP100/files/4993300/Solutionto7B.zip)

**Comment by GraemeMalcolm** [comment]
@rlopezgarcia - I just tested it with SDK 1.11.0. The pipeline completed in just under 25 minutes.
@nish112  - the ParallelRunStep class was moved to the main package (from the conrib package) in SDK 1.10.0, introducing a dependency on azureml-default and azureml-dataprep[fuse], which your updated notebook doesn't address. The lab notebook was updated for this on July 21st.

Couple of possible reasons for the failure you saw:
- Make sure you've pulled the latest version of the repo - we update it constantly as new releases of the SDK are dropped.
- The RunDetails widget sometimes doesn't update - check the experiment run status in Azure ML Studio - it may have finished, but not be reflected in the widget.
- Cloud services are susceptible to transient network issues - if a failure, wait and try again.
